core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -153,7 +153,8 @@ public boolean isSyncPoint(AlluxioURI syncPoint) {
   public boolean isUnderSyncPoint(AlluxioURI path) {
     for (AlluxioURI syncedPath : mSyncPathList) {
       try {
-        if (PathUtils.hasPrefix(path.getPath(), syncedPath.getPath())) {
+        if (PathUtils.hasPrefix(path.getPath(), syncedPath.getPath())
```

Comment:
@yuzhu is a test easy?

Reply:
unfortunately not.. activesync tests are hard because local file system does not support it.. i actually wrote a test and then disabled checks for file system type and it passed.. We need to come up with better ways of testing it.